providers.tf

```diff
@@ -1,3 +1,4 @@
 # Default provider information.
 provider "aws" {
+  region = var.aws_region
```

Comment:
I think we _do_ want to specify the `region` if the code is for a deployment (a la [cisagov/cool-sharedservices-networking](https://github.com/cisagov/cool-sharedservices-networking)) but we _do not_ if it's for an actual Terraform module (a la [cisagov/cool-cdm-cloudtrail-tf-module](https://github.com/cisagov/cool-cdm-cloudtrail-tf-module)).

Since this skeleton is primarily for TF modules (see the name), I think we should remove the `region` in the `provider` blocks.

Reply:
That's a reasonable argument. I think this does highlight that we use this repo for two very different types of Terraform projects however.

Reply:
> Since this skeleton is primarily for TF modules (see the name), I think we should remove the `region` in the `provider` blocks.

I'm with @jsf9k on this one.  What if we comment out the region here and add an explanation about when it might be needed?

Reply:
> I think this does highlight that we use this repo for two very different types of Terraform projects however.

What @dav3r said.  We can comment out the region here and leave a comment that explains the two uses.  We should also add something about this to `README.md`.  I can take a pass at this tomorrow morning if you'd like.

Reply:
I have dropped the commit for this change because implementation of https://github.com/cisagov/skeleton-tf-module/issues/48 would negate its usefulness.